src/yio-plugin/integration.cpp

```diff
@@ -46,7 +46,7 @@ IntegrationInterface::~IntegrationInterface() {}
 Integration::Integration(const QVariantMap& config, EntitiesInterface* entities, NotificationsInterface* notifications,
                          YioAPIInterface* api, ConfigInterface* configObj, Plugin* plugin)
     : QObject(plugin->m_useWorkerThread ? nullptr : plugin),
-      m_state(DISCONNECTED),
+      m_state(States::DISCONNECTED),
```

Comment:
For all enum values: use the full name, e.g. `IntegrationConnectionStates::DISCONNECTED`